java/src/jmri/managers/DefaultIdTagManager.java

```diff
@@ -61,6 +63,7 @@ public void init() {
             dirty = false;
             initShutdownTask();
             initialised = true;
+            propertyChangeSupport.firePropertyChange("PROPERTY_INITLIASED", false, true);
```

Comment:
Shold this be `PROPERTY_INITIALISED` (spelling change, not in quotes)?

Reply:
Definitely, yes! Thanks for the catch.